engine/itb_util_core.py

```
@@ -76,46 +79,44 @@
-    IMPORT_QUEUE_SUCCESSFUL = False
+    import pyaudio as _pyaudio_runtime
+except ImportError:
+    pyaudio: Optional[ModuleType] = None  # pylint: disable=invalid-name
+else:
+    pyaudio = _pyaudio_runtime
 
+langtable: Optional[ModuleType]
 try:
     import langtable # type: ignore
-    IMPORT_LANGTABLE_SUCCESSFUL = True
 except (ImportError,):
```

Ruff (0.15.12)

[warning] A length-one tuple literal is redundant in exception handlers

Replace with `except ImportError`

(B013)